src/views/projectExplorer/index.ts

```diff
@@ -25,6 +25,34 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
     const decorationProvider = new DecorationProvider();
     context.subscriptions.push(
       window.registerFileDecorationProvider(decorationProvider),
+      commands.registerCommand(`vscode-ibmi-projectexplorer.setActiveProject`, async (element?: ProjectExplorerTreeItem) => {
+        if (element) {
+          ProjectManager.setActiveProject(element.workspaceFolder!);
+          this.refresh();
+        } else {
+          const projectItems: QuickPickItem[] = [];
+          const activeProject = ProjectManager.getActiveProject();
+          for (const iProject of ProjectManager.getProjects()) {
+            const state = await iProject.getState();
+            if (state) {
+              const icon = activeProject && activeProject.workspaceFolder === iProject.workspaceFolder ? `$(root-folder)` : `$(symbol-folder)`;
+              projectItems.push({ label: `${icon} ${iProject.getName()}`, description: state.description });
```

Comment:
If there are two workspace folders that have the same name, they will have the same project name. However, you can distinguish them based on the order in which they appear as it will match the order in the file explorer. Or distinguish them based on the description.

The description is from iproj.json.

![image](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/32170854/e6c386e2-3ce2-45c2-8332-f3d85ba80e7f)